coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java

```diff
@@ -206,7 +205,16 @@ else if (isDisapproved(statusChangeEvent)) {
         }
     }
     
+    protected DocumentRouteHeaderValue getDocumentRouteHeaderValue() {
+        return KcServiceLocator.getService(RouteHeaderService.class).getRouteHeader(
+                this.getDocumentHeader().getWorkflowDocument().getDocumentId());
+    }
   
+    protected void setProtocolDocumentToApproveByDefault() {
+        DocumentRouteHeaderValue document = getDocumentRouteHeaderValue();
+        document.getDocumentType().getDefaultApprovePolicy().setPolicyValue(true);
+    }
```

Comment:
@leoherbie yes it is working in 5.2.1
I'm not sure if anything changed with the default blanket approval policy with new rice upgrade.